pkg/interface/chat/src/js/components/lib/invite-search.js

```diff
@@ -49,15 +49,17 @@ export class InviteSearch extends Component {
     let peers = [],
       peerSet = new Set(),
       contacts = new Map;
-    Object.keys(this.props.groups).map(group => {
-      if (this.props.groups[group].size > 0) {
-        let groupEntries = this.props.groups[group].values();
+
+    Object.keys(this.props.permissions).map(group => {
+      if (this.props.permissions[group].who.size > 0) {
+        let groupEntries = this.props.permissions[group].who.values();
         for (let member of groupEntries) {
           peerSet.add(member);
         }
       }
+
       if (this.props.contacts[group]) {
```

Comment:
Wary of letting this component become idiosyncratic. Any way to pass the permissions as a “group” prop?

Reply:
Can do that. Will update the PR tomorrow to just pass the set in as “group”